homeassistant/components/vallox/fan.py

```diff
@@ -51,31 +51,20 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class ValloxFan(FanEntity):
     """Representation of the fan."""
 
+    _attr_should_poll = False
+
     def __init__(self, name, client, state_proxy):
         """Initialize the fan."""
-        self._name = name
         self._client = client
         self._state_proxy = state_proxy
-        self._available = False
         self._state = None
         self._fan_speed_home = None
         self._fan_speed_away = None
         self._fan_speed_boost = None
 
-    @property
-    def should_poll(self):
-        """Do not poll the device."""
-        return False
-
-    @property
-    def name(self):
-        """Return the name of the device."""
-        return self._name
-
-    @property
-    def available(self):
-        """Return if state is known."""
-        return self._available
```

Comment:
These changes are not related to the functionality as described in the PR title

Reply:
Okay, reduced this one to functional change only.